### PR TITLE
Damacus/tidy up tests

### PR DESCRIPTION
**Pull request by damacus**
### Description

Small tidy up of tests and suites

### Issues Resolved

- none

### Contribution Check List

- [ ] All tests pass.
- [x] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

File: .kitchen.dokken.yml [hunk 1]
```diff
@@ -15,14 +15,8 @@ platforms:
   driver:
     image: dokken/amazonlinux
     pid_one_command: /sbin/init
-
-- name: debian-7
```
Comment (tas50) [inline .kitchen.dokken.yml]: pour one out 🍾

File: resources/server_install.rb [hunk 2]
```diff
@@ -30,6 +30,7 @@
 action :install do
   node.run_state['postgresql'] ||= {}
   node.run_state['postgresql']['version'] = new_resource.version
+  puts "HELLO DAN THE VERSION IS #{new_resource.version}"
```
Comment (tas50) [inline resources/server_install.rb]: ?

**Comment by damacus** [comment]
erm debug?

File: resources/server_install.rb (hunk 2 above)
Comment (damacus) [reply resources/server_install.rb]: I've removed it from latest, but I was debugging, some version numbers were... Wrong.

File: libraries/helpers.rb [hunk 3]
```diff
@@ -1,7 +1,6 @@
 #
 # Cookbook:: postgresql
 # Library:: helpers
-# Author:: David Crane (<davidc@donorschoose.org>)
```
Comment (damacus) [inline libraries/helpers.rb]: I've removed David as the Author here, as none of this code is actually his anymore. We've refactored so much of it. That the methods that he wrote no longer exist

**Pull request merged**